### PR TITLE
added properties for lostrid api

### PR DESCRIPTION
**Pull request by dhanendra06**

### CODE REVIEW

File: sandbox/registration-processor-dmz.properties [hunk 1]
```diff
@@ -164,6 +164,10 @@ registration.processor.max.retry=3
 # for search API response consumed by regclient
 mosip.registration.processor.registration.status.external-statuses-to-consider-processed=UIN_GENERATED,REREGISTER,REJECTED,REPROCESS_FAILED	
 
+mosip.registration.processor.lostrid.version=1.0
+mosip.registration.processor.postalcode.req.url=https://dev.mosip.net/v1/masterdata/registrationcenters
+mosip.registration.processor.lostrid.id=mosip.registration.lostrid
```
Comment (vishwa-vyom) [inline sandbox/registration-processor-dmz.properties]: Move this next line to mosip.registration.processor.registration.sync.id

File: sandbox/registration-processor-dmz.properties [hunk 2]
```diff
@@ -164,6 +164,10 @@ registration.processor.max.retry=3
 # for search API response consumed by regclient
 mosip.registration.processor.registration.status.external-statuses-to-consider-processed=UIN_GENERATED,REREGISTER,REJECTED,REPROCESS_FAILED	
 
+mosip.registration.processor.lostrid.version=1.0
```
Comment (vishwa-vyom) [inline sandbox/registration-processor-dmz.properties]: Move this next line to mosip.registration.processor.registration.status.version

File: sandbox/registration-processor-dmz.properties [hunk 3]
```diff
@@ -164,6 +164,10 @@ registration.processor.max.retry=3
...
+mosip.registration.processor.postalcode.req.url=https://dev.mosip.net/v1/masterdata/registrationcenters
```
Comment (vishwa-vyom) [inline sandbox/registration-processor-dmz.properties:170]: for this add a comment describing what is this property used for

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: sandbox/registration-processor-dmz.properties (hunk 3 above)
Comment (dhanendra06) [reply sandbox/registration-processor-dmz.properties:170]: taken care

File: sandbox/registration-processor-dmz.properties (hunk 1 above)
Comment (dhanendra06) [reply sandbox/registration-processor-dmz.properties]: taken care

File: sandbox/registration-processor-dmz.properties (hunk 2 above)
Comment (dhanendra06) [reply sandbox/registration-processor-dmz.properties]: taken care

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**